js/app.js

```diff
@@ -113,7 +135,7 @@ app.controller("MembersController", ['$scope', '$http', function($scope, $http)
     $scope.profile = imgStr.concat(response.id_token.preferred_username);
     $scope.name = response.id_token.preferred_username;//response.userinfo.given_name + " " + response.userinfo.family_name;
   }).error(function (error) { 
-  console.error("Error getting sso");
+    console.error("Error getting sso");
```

Comment:
Debug Statement?

Reply:
I happened to glance over the code and noticed that that console error was not indented correctly. so I added an indent. That's from some much older commit, and wasn't relevant to the meeting stuff.